README.md

```diff
@@ -0,0 +1,47 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Global School Conferance 
+
```

Comment:
- [ ] Please record a 5-minute video presenting the features of the project. Make sure in your video :
 - The video contains a clear and objective description of the project in professional English. The topic and purpose of the project are made clear.
- The video contains a demo of the project, showing the pages in different screen sizes.
- The video contains a highlight about a piece of the code the student finds interesting or is proud of making.